<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findbook</class>
 <widget class="QDialog" name="findbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书查询</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>查询所有书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按类别查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按书名查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按出版社查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按年份查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按作者查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按价格查询</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(expand.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;欢迎使用图书查询功能!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>1000</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="placeholderText">
    <string>请在此输入关键词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快速借阅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快速还书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(cat.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
